pyproject.toml

```
@@ -26,3 +24,9 @@ exclude = '''
 profile = "black"
 combine_as_imports = true
 skip =  ["venv", "benchmarks"]
+
+[build-system]
+requires = ["setuptools>=45", "setuptools_scm[toml]>=6.2", "wheel"]
+
+[tool.setuptools_scm]
+write_to = "unyt/_version.py"
```

I suggest using the `"post-release"` version scheme instead of the default
```suggestion
write_to = "unyt/_version.py"
version_scheme = "post-release"
```

This gives for instance `'2.9.2.post105+g7883d04.d20221010'` instead of `'2.9.3.dev105+g7883d04'`, which I think is preferable because we don't know if the next release is going to be numbered `2.9.3` yet